.pre-commit-config.yaml

```diff
@@ -0,0 +1,25 @@
+repos:
+-   repo: https://github.com/psf/black
+    rev: 19.3b0
```

Comment:
We are using `black==20.8b1` in `setup.py`

https://github.com/huawei-noah/SMARTS/blob/ce1061bba9e7e9f00510443b2ce09a07d9f6ed64/setup.py#L81

Should we use the same black version here?